### PR TITLE
Nintendo Music URLs

### PR DESCRIPTION
**Pull request by PhantomRuby**
Adds URLs to Nintendo Music on applicable songs.

+ minor fixes

### CODE REVIEW

**Comment by PhantomRuby** [comment]
still need to add Pokemon, but that's gonna be quite the undertaking

File: album/pokemon-scarlet-violet-super-music-collection.yaml [hunk 1]
```diff
@@ -705,27 +771,31 @@ Duration: '2:55'
 Artists:
 - Hitomi Sato
 URLs:
+- https://m.nintendo.com/share/US/tracks/199248e3-1367-4095-b83c-f0e11c7038b8
```
Comment (leo60228) [inline album/pokemon-scarlet-violet-super-music-collection.yaml:774]:
```suggestion
- https://m.nintendo.com/shared/US/tracks/199248e3-1367-4095-b83c-f0e11c7038b8
```

Comment (leo60228) [inline album/pokemon-scarlet-violet-super-music-collection.yaml]:
```suggestion
- https://m.nintendo.com/shared/US/tracks/199248e3-1367-4095-b83c-f0e11c7038b8
```

File: album/pokemon-scarlet-violet-super-music-collection.yaml [hunk 2]
```diff
@@ -1803,6 +1991,7 @@ Duration: '5:27'
 Artists:
 - Go Ichinose
 URLs:
+- https://m.nintendo.com/shared//US/tracks/62b941f5-7c93-4d14-b577-a65507c83053
```
Comment (leo60228) [inline album/pokemon-scarlet-violet-super-music-collection.yaml]:
```suggestion
- https://m.nintendo.com/shared/US/tracks/62b941f5-7c93-4d14-b577-a65507c83053
```

**Review by leo60228 — CHANGES_REQUESTED** [review]
URLs all look correct otherwise

**Review by leo60228 — APPROVED** [review]

**Pull request merged**